package.json

```diff
@@ -77,12 +77,12 @@
     "cross-env": "7.0.3",
     "global-jsdom": "9.1.0",
     "husky": "8.0.3",
-    "license-checker-rseidelsohn": "3.3.0",
+    "license-checker-rseidelsohn": "4.2.7",
     "lint-staged": "14.0.1",
     "mocha": "10.2.0",
     "pkgroll": "1.11.0",
     "prettier": "3.0.3",
-    "sinon": "15.2.0",
+    "sinon": "16.0.0",
     "ts-node": "10.9.1",
     "typescript": "5.2.2"
```

Comment:
A lot of these should be devDependencies, then end-users won't end up installing them just to use the package.

Reply:
The `devDependencies` start on line [69](https://github.com/express-rate-limit/rate-limit-postgresql/blob/feature/update-references/package.json#L69) in `package.json`. Is this what you are referring to?

Reply:
🤦 sorry, I missed that.

Reply:
No worries